CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# FDC3 Contribution and Governance Policies
+
+This document describes the contribution process and governance policies of the FINOS FDC3 project. The project is also governed by the [Linux Foundation Antitrust Policy], and the FINOS [IP Policy], [Code of Conduct], [Collaborative Principles], and [Meeting Procedures].
+
+[Linux Foundation Antitrust Policy]: https://www.linuxfoundation.org/antitrust-policy/
+[IP Policy]: https://github.com/finos/community/blob/master/governance/IP-Policy.pdf
+[Code of Conduct]: https://github.com/finos/community/blob/master/governance/Code-of-Conduct.md
+[Collaborative Principles]: https://github.com/finos/community/blob/master/governance/Collaborative-Principles.md
+[Meeting Procedures]: https://github.com/finos/community/blob/master/governance/Meeting-Procedures.md
+
```

Comment:
@mcleo-d @maoo I had to fix these links to the actual FINOS documents - in the templates in the blueprint they lead to 404s.

You may want to update the template to use the full URLs.

Reply:
TY @rikoe ! I've created https://github.com/finos/project-blueprint/issues/20 to investigate further.

Reply:
@rikoe and @maoo - I have raised https://github.com/finos/project-blueprint/pull/21 that fixes the broken links in FINOS blueprint. @maoo is assigned to review.